.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!--- Add a pull request title above in this format -->
+<!--- real example: 'FLAG-51: Replace use of deprecated isVoided' -->
+<!--- 'FLAG-JiraIssueNumber: JiraIssueTitle' -->
+## Description of what I changed
+<!--- Describe your changes in detail -->
+<!--- It can simply be your commit message, which you must have -->
+
+## Issue I worked on
+<!--- This project only accepts pull requests related to open issues -->
+<!--- Want a new feature or change? Discuss it in an issue first! -->
+<!--- Found a bug? Point us to the issue/or create one so we can reproduce it! -->
+<!--- Just add the issue number at the end: -->
+see https://issues.openmrs.org/browse/FLAG-
```

Comment:
Change this to https://openmrs.atlassian.net/browse/FLAG-

Reply:
I updated the JIRA issue url